### PR TITLE
bugfix/auc

### PR DESCRIPTION
**Pull request by bkleyn**
Bugfix for AUC metric to deal with situation where only one class is present in actual responses making AUC undefined.

### CODE REVIEW

**Comment by dorukkilitcioglu** [comment]
What is the current behavior? Does it raise an error?

**Comment by bkleyn** [comment]
> What is the current behavior? Does it raise an error?

Yes, raises error.

File: jurity/recommenders/auc.py [hunk 1]
```diff
@@ -142,7 +142,10 @@ def get_score(self, actual_results: pd.DataFrame, predicted_results: pd.DataFram
     @staticmethod
     def _get_results(results: List[np.ndarray]) -> float:
         results = np.concatenate(results)
-        return roc_auc_score(results[:, 0], results[:, 1])
+        if len(set(results[:, 0])) > 1:
```
Comment (filip-michalsky) [inline jurity/recommenders/auc.py:145]: do we guarantee `results` is a 2D array somewhere?

**Review by dorukkilitcioglu — APPROVED** [review]
LGTM! The way these metrics are set up, even if you get a `nan` in your first batch of evaluations, you should still be able to get a correct value in the second batch. So I don't see `nan` causing any numerical problems within Jurity - we just have to be cognizant of it when we're using Jurity.

**Comment by filip-michalsky** [comment]
Looks like I can't give a second approval for some reason... good to got from my side!

**Pull request merged**